neural_lam/models/ar_model.py

```diff
@@ -432,6 +498,14 @@ def test_step(self, batch, batch_idx):
         self.spatial_loss_maps.append(log_spatial_losses)
         # (B, N_log, num_grid_nodes)
 
+        if self.args.save_eval_to_zarr_path:
+            self._save_predictions_to_zarr(
+                batch_times=batch_times,
+                batch_predictions=prediction,
```

Comment:
These predictions are in the standardized scale. At some point before these are written to disk in the zarr they should be rescaled to the original data scale.

Reply:
Can be done as https://github.com/joeloskarsson/neural-lam-dev/pull/9/commits/d3f636e9ec573b063ed0fcd236f572c5dfe0a5b7